src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
Please merge this file with the existing IntakeSubsystem and delete this one.
We keep one intake subsystem.